test/unit/commands-test.js

```diff
@@ -42,19 +42,35 @@ var config = require('./testConfig'),
     ngsiClient = ngsiTestUtils.create(
         config.ngsi.contextBroker.host,
         config.ngsi.contextBroker.port,
-        'smartGondor',
-        '/gardens'
+        'dumbmordor',
```

Comment:
Not sure about this change... In my opinion it should be just

smartGondor -> smartgondor

Why to change to something completely different (dumbmordor)? Why does subservice needs also to be changed?

Reply:
Fixed in some recent commit?

Reply:
Fixed in babe9842b3d75a735f68117dcbefa46228bacf55